dartagnan/src/main/java/com/dat3m/dartagnan/verification/solving/ModelChecker.java

```diff
@@ -109,7 +109,7 @@ public static void performStaticWmmAnalyses(VerificationTask task, Context analy
         analysisContext.register(RelationAnalysis.class, RelationAnalysis.fromConfig(task, analysisContext, config));
     }
 
-    protected void saveFlaggedPairsOutput(Wmm wmm, WmmEncoder encoder, ProverEnvironment prover, EncodingContext ctx, Program program) throws SolverException {
+    protected void saveFlaggedPairsOutput(Wmm wmm, WmmEncoder encoder, ProverWithTracker prover, EncodingContext ctx, Program program) throws SolverException {
```

Comment:
This is one of the cases where the use-site could keep working on `ProverEnvironment` if it was implemented.